ClientApp/src/Pages/Conservation/Conservation.tsx

```diff
@@ -0,0 +1,94 @@
+import "./Conservation.scss";
+import "bootstrap";
+import React from "react";
+import * as ConservationData from "./ConservationData.json";
+
+export function Conservation(): JSX.Element {
+  return (
+    <div>
+
+      <div className="container-fluid">
+          <h2 className="conservationTitle">Conservation</h2>
+          <p className="text-justify">
+            Some good work has been done in recent decades to reverse the trend of
+            almost inevitable extinction for some species of cetaceans.
+            Unfortunately, whales and dolphins continue to be threatened by:
+          </p>
+      </div>
+
+      <div className="container">
+
+        <div className="row">
+
+          <div className="col-sm">
+            <div className="card">
+              <img src={ConservationData.Pollution.imgSrc} />
```

Comment:
In general all `img` tags need some alt text so that they can be understood even if you can't see the image.
Do you think you could add one here?

Reply:
img tags now have alt values